taskrun/TaskManager.py

```diff
@@ -223,6 +223,25 @@ def task_failed(self, task, errors):
     This function is called by a task when an error code is returned from
     the task
 
+    Args:
+      task (Task) : the task that encountered errors
+    """
+    self._task_failed_or_killed(task, errors)
+
+  def task_killed(self, task):
+    """
+    This function is called by a task when it has been killed
+
+    Args:
+      task (Task) : the task that was killed
+    """
+    self._task_failed_or_killed(task, None)
+
+  def _task_failed_or_killed(self, task, errors):
```

Comment:
_task_errored?

Reply:
Hmmm, "task errored" makes it sound like it was its fault, but that isn't the case for killed. How about _task_error?

Reply:
SGTM